memorywipe.py

```diff
@@ -0,0 +1,496 @@
+                13: "serpent(aes)",
+                14: "serpent(twofish(aes))",
+                15: "twofish(serpent)"
+                }
+        encrypt_type = encrypt_type_dict[click.prompt("Select", default=1, type=click.IntRange(1, 15), show_choices=False)]
+        click.echo("Hash algorithm:\n1) SHA-512\n2) Whirlpool\n3) BLAKE2s-256\n4) SHA-256\n5) Streebog\n")
+        hash_algo_dict = {1: "sha-512", 2: "whirlpool", 3: "blake2s-256", 4: "sha-256", 5: "streebog"}
+        hash_algo = hash_algo_dict[click.prompt("Select", default=1, type=click.IntRange(1, 5), show_choices=False)]
+        click.echo(
+            "Filesystem:\n"
+            "1) None\n"
+            "2) FAT\n"
+            "3) Linux Ext2\n"
+            "4) Linux Ext3\n"
+            "5) Linux Ext4\n"
+            "6) NTFS"
+            )
+        filesystem_dict = {
+            1: "none",
+            2: "fat",
+            3: "linux ext2",
+            4: "linux ext3",
+            5: "linux ext4",
+            6: "ntfs"
+        }
+        filesystem = filesystem_dict[click.prompt("Select", default=2, type=click.IntRange(1, 6), show_choices=False)]
+        click.echo("\nEnter strong password for encrypting. (You don't have to remember it)")
+        strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
+        out = subprocess.run(["sudo", "veracrypt", "-tc", f"--volume-type={vol_type}", 
+                                f"{self.partition}", f'--encryption={encrypt_type}', f'--hash={hash_algo}', 
+                                f"--filesystem={filesystem}", "-p", f"{strongp}", "--pim=0", 
+                                "-k", "", "--random-source=/dev/urandom"], capture_output=True)
+        if out.returncode:
+            click.secho(out.stderr.decode(), fg="yellow")
+        show_out = click.prompt("Do you wish to see the output of the command run?", type=click.Choice(["y", "n"]), default="n")
+        if show_out == "y":
+            click.echo(out.stdout.decode())
+                  
+    def _veracrypt_encrypt(self):
+        if self.chk_encrypted():
+            click.echo("Selected partition is already encrypted! Skipping encryption...\n")
+        else:
+            q2 = click.prompt("Do you want to encrypt manually or automatically?", type=click.Choice(["m", "a"]), show_choices=True, default="a")
+            if q2 == "m":
+                click.echo("Starting veracrypt manually...\n")
+                self._veracrypt_interactive()
+                click.secho(f"Finished encrypting {self.partition}\n", fg="green")
+            elif q2 == "a":
+                click.echo("Uses AES, with SHA-512 and makes an NTFS filesystem")
+                click.echo("\nEnter strong password for encrypting. (You don't have to remember it)")
+                strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
```

Comment:
The password set here should be random and strong, and the user doesn't have to remember it. So, set the confirmation_prompt to false here.

Reply:
This is a simple fix, so I will edit it myself.